app/controllers/change_requests_controller.rb

```diff
@@ -13,15 +13,26 @@ def create
   end
 
   def create_change_request
+    puts "#{params}"
```

Comment:
Please remove all puts that are not necessary for the ongoing operation of the system. Statements like this without context will clutter logs and do not provide long term use.